benefits/in_person/views.py

```diff
@@ -137,6 +155,7 @@ def enrollment(request):
 
 def reenrollment_error(request):
     """View handler for a re-enrollment attempt that is not yet within the re-enrollment window."""
+    enrollment_analytics.returned_error(request, "Re-enrollment error.", enrollment_method=models.EnrollmentMethods.IN_PERSON)
```

Comment:
Could be moved up into `POST`-back

Reply:
Yep, moved it up in f5a376abb9e483dcd8b7f50ab34dedd24ff6ffb6